gateway/src/apicast/executor.lua

```diff
@@ -52,7 +52,7 @@ end
 -- @tparam string phase Nginx phase
 -- @treturn linked_list The context. Note: The list returned is 'read-write'.
 function _M:context(phase)
-    if phase == 'init' then
+    if phase == 'init' or phase == 'ssl_certificate' then
```

Comment:
why?

Reply:
Good question. That is probably a leftover from a debugging session. Will take a look.